yt/geometry/coordinates/cartesian_coordinates.py

```diff
@@ -369,20 +369,20 @@ def _ortho_pixelize(self, data_source, field, bounds, size, antialias,
                     for chunk in data_source.chunks([], 'io'):
                         pixelize_sph_kernel_slice(
                             buff,
-                            chunk[ptype, px_name],
-                            chunk[ptype, py_name],
-                            chunk[ptype, 'smoothing_length'],
-                            chunk[ptype, 'particle_mass'],
+                            chunk[ptype, px_name].to('code_length'),
```

Comment:
Would it be better to use `to_value` in these cases? Or probably it doesn't matter.

Reply:
I don't think it does matter, though I note that the other arrays have units.

Reply:
I guess it works because the implicit type conversion from `unyt` to `numpy` array won't change the array value.